features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1899,6 +1899,17 @@
 
     <EnableSessionNonceCookie>{{session.nonce.cookie.enabled}}</EnableSessionNonceCookie>
 
+    {% if session.nonce.cookie.default_whitelist_authenticators is defined %}
```

Comment:
Is it possible to check if either default is defined or the custom one is defined? and not just check if the default is defined. 

Otherwise, we need to remove this condition here. Else if there are no defaults and there are user defined ones then the congif will not apply